src/components/LocaleSelector/LocaleSelector.astro

```diff
@@ -1,40 +1,46 @@
 ---
-import { getLocale, locales, t } from '@lib/i18n';
+import { getLocale, getLocaleName, locales, t } from '@lib/i18n';
```

Comment:
Not really part of this PR. But implementing the `FileLink` I discovered the `Intl.DisplayNames` API, which led me to use it here and improve the accessibility of the locale selector a bit while going at it.